cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -149,15 +152,13 @@ Future<Void> reconcile(ReconciliationState reconcileState)  {
                 // Preparation steps => prepare cluster descriptions, handle CA creation or changes
                 .compose(state -> state.reconcileCas(this::dateSupplier))
                 .compose(state -> state.versionChange())
-
```

Comment:
I think this empty line was intentional to better separate the different parts ;-)

Reply:
Sorted :+1: